crates/templates/src/manager.rs

```diff
@@ -121,6 +123,7 @@ impl TemplateManager {
         source: &TemplateSource,
         options: &InstallOptions,
         reporter: &impl ProgressReporter,
+        install_template_list: &Vec<String>,
```

Comment:
Is there a reason for this to be a separate parameter rather than part of `InstallIOptions`?

Reply:
Nothing more than me just hacking something quickly, I will move it as a part of the `InstallOptions`.